specification/sdk-exporter.md

```diff
@@ -0,0 +1,91 @@
+# Exporter Interface
+
+Exporter defines the interface that protocol-specific exporters must implement so that they can be plugged into OpenTelemetry SDK and support sending of telemetry data.
+
+The goals of the interface are:
+
+- Minimize burden of implementation for protocol-dependant telemetry exporters. The protocol exporter is expected to be primarily a simple telemetry data encoder and transmitter.
+
+- Allow implementing helpers as composable components that use the same chainable Exporter interface. SDK authors are encouraged to implement common functionality such as queuing, batching, tagging, etc. as helpers. This functionality will be applicable regardless of what protocol exporter is used.
+
+## Interface Definition
+
+The exporter must support two functions: **Export** and **Shutdown**. In strongly typed languages typically there will be 2 separate Exporter interfaces, one that accepts Spans (SpanExporter) and one that accepts Metrics (MetricsExporter).
```

Comment:
We may need a flush as well.

Reply:
If we are not yet sure, let's keep as is. We can add flush later.